packages/forms-web-app/src/routes/pdf/index.js

```diff
@@ -0,0 +1,39 @@
+const axios = require('axios');
+const FormData = require('form-data');
+const express = require('express');
+const { getExistingAppeal } = require('../../lib/appeals-api-wrapper');
+const { getDepartmentFromId } = require('../../services/department.service');
+const { VIEW } = require('../../lib/views');
+
+const router = express.Router();
+
+router.get('/source/:id', async (req, res) => {
+  const appeal = await getExistingAppeal(req.params.id);
+
+  let appealLPD = '';
+
+  if (appeal.lpaCode) {
+    const lpd = await getDepartmentFromId(appeal.lpaCode);
+    if (lpd) {
+      appealLPD = lpd.name;
+    }
+  }
+
+  res.render(VIEW.PDF.SOURCE, {
+    appealLPD,
+    appeal,
+  });
+});
+
+router.get('/download/:id', async (req, res) => {
```

Comment:
2nd pdf endpoint that does the download; it gives the ^endpoint to gotenburg and gets it to render the pdf... returns the pdf as a stream for download

again; not safe. happy-path.